spdx-utils/src/main/kotlin/model/SpdxPackage.kt

```diff
@@ -199,12 +200,16 @@ data class SpdxPackage(
 
         val validPrefixes = listOf(SpdxConstants.PERSON, SpdxConstants.ORGANIZATION)
 
-        require(originator.isEmpty() || validPrefixes.any { originator.startsWith(it) }) {
-            "A non-empty originator has to start with any of $validPrefixes."
+        if (originator != null) {
+            require(originator == SpdxConstants.NOASSERTION || validPrefixes.any { originator.startsWith(it) }) {
```

Comment:
Consider `require(originator != null || ...)` to reduce indentation and to save 2 lines.

Reply:
That was my initial preference, too, but that makes the line too long and thus requires a line break, so I went for this approach.